modules/spring-config/src/main/resources/default-mule-config-override.xml

```diff
@@ -2,10 +2,7 @@
 <beans xmlns="http://www.springframework.org/schema/beans"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="
-       http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-current.xsd"
-       default-init-method="initialise">
-
-    <bean name="_muleSimpleRegistryBootstrap" class="java.lang.Object"/>
```

Comment:
This object override was added in order to avoid package scanning when calculating metadata from Studio. If this is removed, do that means that the SpringRegistryBootstrap is going to be used?